doc/lua_api.txt

```diff
@@ -6763,6 +6763,18 @@ an itemstring, a table or `nil`.
 * `peek_item(n)`: returns taken `ItemStack`
     * Copy (don't remove) up to `n` items from this stack
     * `n`: number, default: `1`
+* `equals(other)`:
+    * returns `true` if this stack is identical to `other`.
+    * Note: `stack1:to_string() == stack2:to_string()` is not reliable,
+      as stack metadata can be serialized in arbitrary order.
+
+Operators
```

Comment:
Shouldn't this be `### Operators` so that it is placed in the `ItemStack` section?

Reply:
i dunno; i'm borrowing the syntax for an `operators` section from https://github.com/minetest/minetest/blob/500ebc859d90ad51e366530d8ca01ab0cfe5425f/doc/lua_api.txt#L3511-L3512  

easy to fix, if someone confirms way it should be.

Reply:
There "Operators" (header level 2) is a subsection of "Spatial Vectors" (header level 1). Here "Operators" should be header level 3, as it is a subsection of "`ItemStack`" (header level 2).